gen/ir/constructors.go

```diff
@@ -73,3 +79,22 @@ func Stream(name string, schema *jsonschema.Schema) *Type {
 		Schema: schema,
 	}
 }
+
+func External(schema *jsonschema.Schema) (*Type, error) {
+	i := strings.LastIndex(schema.XOgenType, ".")
+	if i < 0 || i == len(schema.XOgenType)-1 {
+		return nil, fmt.Errorf("'%s' is not a valid Go type (expected 'pkg/path.Type')", schema.XOgenType)
```

Comment:
We usually use go-faster/errors
```suggestion
		return nil, errors.Errorf("%q is not a valid Go type (expected 'pkg/path.Type')", schema.XOgenType)
```

Reply:
Done.